CHANGELOG.md

```diff
@@ -6,6 +6,14 @@ NOTE: [`epaint`](crates/epaint/CHANGELOG.md), [`eframe`](crates/eframe/CHANGELOG
 
 ## Unreleased
 
+### ⭐ Added
+
+- Plot images can now be rotated [#3182](https://github.com/emilk/egui/pull/3182)
+
+### 🔧 Changed
+
+- The scale of plot images is now in plot coordinates instead of screen coordinates [#3182](https://github.com/emilk/egui/pull/3182)
+- Plot polygons can now have a custom fill color, replaced `Polygon::fill_alpha` and `Polygon::color` with `Polygon::fill_color` [#3182](https://github.com/emilk/egui/pull/3182)
```

Comment:
```suggestion
```

I've switched to generating the changelog at release-time using `scripts/generate_changelog.py`.

Please add the above details to the PR description instead!